src/scripts/theme-toggle-persistence.js

```diff
@@ -11,28 +11,28 @@
  */
 
 export function initThemeTogglePersistence() {
+
+  const themeToggleWrapper = document.getElementById('theme-toggle');
```

Comment:
make sure all the elements we are using are there. don't query document, element queries have less html tree to traverse.